### ISSUE TITLE
Run app

### ISSUE THREAD

**Issue by mrriviere** [Bartozzz/Qilin#27]
Forgive me the ignorance but, how can i run this app on ubuntu?

**Comment by Bartozzz** [comment]
**The easy way:**
You can download one of the [old releases](https://github.com/Bartozzz/Qilin/releases) (I believe there should be a compressed package for Linux, pheraps `qilin-lin.zip`). I never used Ubuntu, but it should work on most Linux distributions.

**The better way:**
I recommend you building Qilin yourself to get the latest features (the last release is about 30 commits behind the `master` branch). You will need [Node.js](https://nodejs.org/en/download/) installed locally on your machine and follow [instructions in README.md](https://github.com/Bartozzz/Qilin#development):

1. Clone or [download](https://github.com/Bartozzz/Qilin/archive/master.zip) Qilin's repository from GitHub;
2. Move to the downloaded directory in your terminal (`$ cd path/to/Qilin`);
3. Download NPM dependencies via: `$ sudo npm i`;
4. Once all dependencies are installed, build Qilin via: `sudo npm run build`;
5. Once the build is finished, you can find Qilin binaries in `build/` folder;

Let me know if it works for you.

**Comment by mrriviere** [comment]
I tried with his steps, but the build crashes and generate this log file. Please, can you help me?

`<script src="https://gist.github.com/mrriviere/4b4f3e00ccc233992be3a7bece2f2a40.js"></script>`

**Comment by mrriviere** [comment]
I tried with his steps, but the build crashes and generate this log file. Please, can you help me?
<script src="https://gist.github.com/mrriviere/4b4f3e00ccc233992be3a7bece2f2a40.js"></script>

**Comment by Bartozzz** [comment]
Are you sure you downloaded all the _npm dependencies_ (step 3) and ran the _build script_ (step 4) **with administrator privileges** (`sudo`)? Your log file isn't showing any precise informations about why the build is failing.

Can you try the following steps, so I can see at which one the build breaks?

1. Execute `sudo npm run prepare` in the repository root;
2. Execute `sudo node ./bin/compile` in the repository root;

**Comment by mrriviere** [comment]
Thank's a lot!! The problem was with administrator privileges indeed. I fixed it and I hope contribute with this project soon.

**Comment by mrriviere** [comment]
Oh a new issue appears. I can't see the toolbar on the left. It was discontinued?

**Comment by Bartozzz** [comment]
Yes, indeed. Left toolbar contained anchors to **formula mode** and **sketch mode**. [Sketch mode](http://i.imgur.com/GF2xiYT.jpg) was abandonned because it was a pain to maintain, so I focused on developping the main editor (_keep it simple, stupid_). Formula mode wasn't finished yet anyway.

Right now, I am planning to create a pop-up on-top of Qilin editor to help people write forumlas (something similar to Wikiedia LaTeX editor) instead of a totally separate view which was covering all the text:

<img width="901" alt="zrzut ekranu 2017-09-02 o 01 18 55" src="https://user-images.githubusercontent.com/6741707/29990631-bcac8b5c-8f7c-11e7-8903-9b2b09edb9be.png">

Feel free to contribute! I'll close this issue as the problem was solved. If you will encounter other issues, don't hesitate to create another one.
